data_steward/cdr_cleaner/cleaning_rules/remove_ehr_data_past_deactivation_date.py

```diff
@@ -41,13 +41,16 @@ def remove_ehr_data_queries(project_id, ticket_number, pids_project_id,
     ehr_union_dataset = bq_utils.get_unioned_dataset_id()
```

Comment:
The only suggestion I have is to update with some comments.  We were both a little confused about `project_id` vs `pids_project_id` and what should be passed to which function.  The docstring doesn't distinguish between the two well enough to allow someone to read it and just know what is supposed to happen.  I can't select that area of the file to comment on, ugh.

Reply:
I added comments and updated the docstring like you suggested